### DISCUSSION TITLE
Show and Tell

### DISCUSSION THREAD

**Discussion by tom-huntington** [ottomated/create-o7-app#7] [Show and tell]
If there are any public github repos using the o7 stack/template, please post them here so I can plunder them :smile:

**Edit**: You can find all the public create-o7-apps here https://github.com/ottomated/trpc-svelte-query/network/dependents

**Comment by tom-huntington** [comment]
There's an example project in the [trpc-svelte-query](https://github.com/ottomated/trpc-svelte-query) repo.

**Reply by tom-huntington** [reply]
Also there's a stackblitz link in this issue https://github.com/sveltejs/kit/issues/9355

**Comment by JewishBidoof** [comment]
I'm working on a link-in-bio site with a postgres connection for dynamic link mgmt. I'll be sure to come back when I have a completed public version, but I have v1 up and running at https://pupdad.xyz

**Reply by JewishBidoof** [reply]
As promised: https://github.com/TrekkerStudios/THBioLink
